ckanext/stcndm/plugins.py

```diff
@@ -79,3 +81,34 @@ def get_helpers(self):
         return {
             "codeset_choices": helpers.codeset_choices,
         }
+
+    def before_view(self, pkg_dict):
+        """
+        Ensure that (if available) the correct language strings
+        are used for core CKAN fields.
+        """
+        default_language = u'en'
+        fields_to_fluent = (
+            u'title',
+            u'notes',
+            u'name',
```

Comment:
'name' in the dataset is the unique url slug, and will never be a dict.

'name' in the resources list, however, might need to be translated